src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,157 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size = 0;
+    private Node<T> first = null;
+    private Node<T> last = null;
```

Comment:
```suggestion
    private int size;
    private Node<T> first;
    private Node<T> last;
```
those fields have those values by default